### DISCUSSION TITLE
OTEL Getting Started document - Meter Provider vs Metric Provider

### DISCUSSION THREAD

**Discussion by tech4gk** [open-telemetry/opentelemetry-go#2088] [General]
Hi,
 I am new to OTEL world. I was going through the [OTEL Go getting started](https://opentelemetry.io/docs/go/getting-started/#creating-a-meter-provider) material and found a reference to Meter Provider. I guess, this is supposed to be a Metric Provider, isn't it ?

Thanks
G

**Comment by MrAlias** [comment]
No, Meter Provider is correct. For more info on the metrics signal programming model be sure to check [this](https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/metrics#programming-model) out.

Also, it's important to understand the metrics signal is still very experimental so terms may change in the future.
